_includes/footer.html

```diff
@@ -38,36 +38,39 @@
 
 .buttonRed:hover {
 
-  background-color: white; 
-  color: black;   
+  background-color: white;
+  color: black;
   border: 2px solid #830506;
 }
 </style>
 <footer class="footer">
     <div class="container">
+
+<!--TODO FIND BETTER ALTERNATIVE TO FORMSUBMIT.CO-->
```

Comment:
Please remove this commented section and create a new GitHub issue to replace this with something that works better.